confgureStore.js

```diff
@@ -0,0 +1,34 @@
+import { createStore, applyMiddleware } from "redux";
+import { createLogger } from "redux-logger";
+import thunk from "redux-thunk";
+import { composeWithDevTools } from "redux-devtools-extension";
+import { persistStore, persistReducer } from "redux-persist";
+import { AsyncStorage } from "react-native";
+
+import rootReducer from "./reducers/rootReducer";
+
+export const INITIAL_STATE = {
+  tasks: {}
+};
+
+const persistConfig = {
+  key: "root",
+  storage: AsyncStorage
+};
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
```

Comment:
Cool, haven't worked with redux-persist before. Seems like this would give us 'out of the box' support for offline read-mode?

Not necessary for the scope of this demo, but a question to consider: if we have redux-persist as our persistent (including offline) storage of tasks, what other steps would we need to take in order to ensure actions taken while offline get properly synced when reconnecting?

Reply:
I have actually a bunch of ideas around how this could be feasible and worked out. Excited to share in our meeting